modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -1,114 +1,80 @@
-A transaction domain is a collection of graphs that can be updated within the context of a single transaction.
-
-Execution context::
-An execution context is a runtime environment for the execution of a request.
-In practical terms, a request may be a query, a transaction, or an internal function or procedure.
-
-Graph::
-A data model within a database.
-
-Database::
 A database is an administrative partition of a DBMS.
 In practical terms, it is a physical structure of files organized within a directory or folder, that has the same name of the database.
-+
-In Neo4j {neo4j-version} each standard database contains a single graph.
+This chapter describes how to manage local and remote standard databases, composite databases, and database aliases.
+
+== Standard databases
+
+In Neo4j {neo4j-version}, each standard database contains a single graph.
 Many administrative commands refer to a specific graph by using the database name.
-+
-A database defines a _transaction domain_ and an _execution context_.
+
+A database defines a _transaction domain_ (a collection of graphs that can be updated within the context of a single transaction) and an _execution context_(a runtime environment for the execution of a request).
 This means that a transaction cannot span across multiple databases.
 Similarly, a procedure is called within a database, although its logic may access data that is stored in other databases.
-+
-A default installation of Neo4j {neo4j-version} contains two databases:
-+
-* `system` - xref:manage-databases/introduction.adoc#manage-databases-system[the system database], containing metadata on the DBMS and security configuration.
-* `neo4j` - xref:manage-databases/introduction.adoc#manage-databases-default[the default database], a single database for user data.
-This has a default name of `neo4j`.
+
+=== Standard databases per Neo4j edition
+
+The edition of Neo4j determines the number of possible databases:
+
+* Installations of Community Edition can have exactly *one* standard database.
+* Installations of Enterprise Edition can have any number of standard databases.
+
+[[manage-databases-default]]
+=== Default database
+
+A default installation of Neo4j {neo4j-version} contains one standard database, named `neo4j`, which is the default database for the DBMS.
 A different name can be configured before starting Neo4j for the first time.
+For details, see xref:database-administration/standard-databases/configuration-parameters.adoc[Configuration parameters].
+
+The following image illustrates an installation of Neo4j containing the three standard databases, named `marketing`, `sales`, and `hr`, and the `system` database.
+The default database is `sales`:
+
+image::manage-dbs-default.png[title="A multiple database Neo4j installation, with a default database.", role="middle"]
 
 [NOTE]
 ====
 Be aware that the automatically created _initial_ default database may have a different topology to the default configuration values.
```

Comment:
shouldn't this note be before the image when you are talking about setting different default database?

Reply:
We will think about it.